server/channels/app/authorization.go

```diff
@@ -68,31 +68,22 @@ func (a *App) SessionHasPermissionToTeams(c request.CTX, session model.Session,
 			return false
 		}
 	}
-	if session.IsUnrestricted() {
+
+	// Check session permission, if it allows access, no need to check teams.
+	if a.SessionHasPermissionTo(session, permission) {
```

Comment:
Just for my understanding, will this check cover the local mode sessions that `session.IsUnrestricted()` handles?

Reply:
Yep. `SessionHasPermissionTo` does that check.